acorn/dist/acorn.d.ts

```diff
@@ -1,209 +1,4 @@
-    isAssign: boolean
-    prefix: boolean
-    postfix: boolean
-    binop: number
-    updateContext?: (prevType: TokenType) => void
-    constructor(label: string, conf?: any)
-  }
+import * as Acorn from './acorn.mjs'
 
-  const tokTypes: {
-    num: TokenType
-    regexp: TokenType
-    string: TokenType
-    name: TokenType
-    eof: TokenType
-    bracketL: TokenType
-    bracketR: TokenType
-    braceL: TokenType
-    braceR: TokenType
-    parenL: TokenType
-    parenR: TokenType
-    comma: TokenType
-    semi: TokenType
-    colon: TokenType
-    dot: TokenType
-    question: TokenType
-    arrow: TokenType
-    template: TokenType
-    ellipsis: TokenType
-    backQuote: TokenType
-    dollarBraceL: TokenType
-    eq: TokenType
-    assign: TokenType
-    incDec: TokenType
-    prefix: TokenType
-    logicalOR: TokenType
-    logicalAND: TokenType
-    bitwiseOR: TokenType
-    bitwiseXOR: TokenType
-    bitwiseAND: TokenType
-    equality: TokenType
-    relational: TokenType
-    bitShift: TokenType
-    plusMin: TokenType
-    modulo: TokenType
-    star: TokenType
-    slash: TokenType
-    starstar: TokenType
-    _break: TokenType
-    _case: TokenType
-    _catch: TokenType
-    _continue: TokenType
-    _debugger: TokenType
-    _default: TokenType
-    _do: TokenType
-    _else: TokenType
-    _finally: TokenType
-    _for: TokenType
-    _function: TokenType
-    _if: TokenType
-    _return: TokenType
-    _switch: TokenType
-    _throw: TokenType
-    _try: TokenType
-    _var: TokenType
-    _const: TokenType
-    _while: TokenType
-    _with: TokenType
-    _new: TokenType
-    _this: TokenType
-    _super: TokenType
-    _class: TokenType
-    _extends: TokenType
-    _export: TokenType
-    _import: TokenType
-    _null: TokenType
-    _true: TokenType
-    _false: TokenType
-    _in: TokenType
-    _instanceof: TokenType
-    _typeof: TokenType
-    _void: TokenType
-    _delete: TokenType
-  }
-
-  class TokContext {
-    constructor(token: string, isExpr: boolean, preserveSpace: boolean, override?: (p: Parser) => void)
-  }
-
-  const tokContexts: {
-    b_stat: TokContext
-    b_expr: TokContext
-    b_tmpl: TokContext
-    p_stat: TokContext
-    p_expr: TokContext
-    q_tmpl: TokContext
-    f_expr: TokContext
-  }
-
-  function isIdentifierStart(code: number, astral?: boolean): boolean
-
-  function isIdentifierChar(code: number, astral?: boolean): boolean
-
-  interface AbstractToken {
-  }
-
-  interface Comment extends AbstractToken {
-    type: string
-    value: string
-    start: number
-    end: number
-    loc?: SourceLocation
-    range?: [number, number]
-  }
-
-  class Token {
-    type: TokenType
-    value: any
-    start: number
-    end: number
-    loc?: SourceLocation
-    range?: [number, number]
-    constructor(p: Parser)
-  }
-
-  function isNewLine(code: number): boolean
-
-  const lineBreak: RegExp
-
-  const lineBreakG: RegExp
-
-  const version: string
-}
+export as namespace acorn
```

Comment:
I might be mistaken, but I don't think namespace was exported before, it was used only for the `export =`.

Reply:
Oh wait, I see it was...